macros/commands/enforce_project_configurations.sql

```diff
@@ -0,0 +1,66 @@
+{%- macro enforce_project_configurations(enforce_owners=true, enforce_tags=false, enforce_meta_params=[], enforce_config_params=[], exclude_sources=false) -%}
+    {%- if execute -%}
+        {# enforcing source params #}
+        {%- if not exclude_sources -%}
+            {% set sources = graph.sources.values() | selectattr('resource_type', '==', 'source') %}
+            {% set sources_result = elementary.enforce_configuration(sources, elementary.flatten_source, enforce_owners, enforce_tags, enforce_meta_params, enforce_config_params) %}
+        {%- endif -%}
+
+        {# enforcing model params #}
+        {% set models = graph.nodes.values() | selectattr('resource_type', '==', 'model') %}
+        {% set models_result = elementary.enforce_configuration(models, elementary.flatten_model, enforce_owners, enforce_tags, enforce_meta_params, enforce_config_params) %}
+
+        {%- if not models_result or not sources_result -%}
+            {{ exceptions.raise_compiler_error("Found issues in projdct configurations") }}
```

Comment:
```suggestion
            {{ exceptions.raise_compiler_error("Found issues in project configurations") }}
```

Reply:
This would always raise if `exclude_sources`.
Also, I'd consider changing to `include_sources=true`.
I think it's easier comprehend than negating, kind of like us having `disable` configurations rather than `enable`.